app/src/main/java/edu/uc/kovaciad/slicetracker/MainActivity.kt

```diff
@@ -103,7 +105,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     // On swipe right, go back to main page
-    private fun onSwipeRight() {
+    internal fun onSwipeRight() {
```

Comment:
Changing from private to internal as a function that navigates between fragments could be used in other classes. Example of this can be found in instructors MainActivity file here:https://github.com/discospiff/MyPlantDiaryQ/blob/RecyclerViewAndSuperclassFragment/app/src/main/java/edu/uc/jonesbr/myplantdiary/MainActivity.kt

Reply:
Did you make any changes to use it in other classes?  If not, I would leave the visibility as-is.  No need to give grander scope that is not being used.